### PR TITLE
docs(apigateway): fix sample layout provided

### PR DESCRIPTION
**Pull request by michaelbrewer**
Current version does not work when deployed

**Issue #, if available:**

- https://github.com/awslabs/aws-lambda-powertools-python/issues/835

## Description of changes:

Fixes the sample layout for multiple file project

**Checklist**

 

* [X] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [X] Update tests
* [X] Update docs
* [X] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/376ec0a2ac0d2a40e0af5717bef42ff84ca0d1b9/.github/semantic.yml#L2)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

-----
[View rendered docs/core/event_handler/api_gateway.md](https://github.com/gyft/aws-lambda-powertools-python/blob/doc-updated-layout/docs/core/event_handler/api_gateway.md)

### CODE REVIEW

File: docs/core/event_handler/api_gateway.md [hunk 1]
```diff
@@ -1073,6 +1072,8 @@ This sample project contains an Users function with two distinct set of routes,
             MemorySize: 512
             Runtime: python3.9
             Tracing: Active
+            Architectures:
+                - arm64
```
Comment (heitorlessa) [inline docs/core/event_handler/api_gateway.md:1076]: We build Layer for x86 only atm

File: docs/core/event_handler/api_gateway.md [hunk 2]
```diff
@@ -1027,43 +1027,42 @@ When necessary, you can set a prefix when including a router object. This means
...
+This sample project contains a Users function with two distinct set of routes, `/users` and `/health`. The layout optimizes for code sharing, no custom build tooling, and it uses [Lambda Layers](../../index.md#lambda-layer) to install Lambda Powertools.
 
 === "Project layout"
 
 
-    ```python hl_lines="6 8 10-13"
+    ```python hl_lines="1 8 10 12-15"
     .
-    ├── Pipfile                   # project app & dev dependencies; poetry, pipenv, etc.
+    ├── Pipfile                    # project app & dev dependencies; poetry, pipenv, etc.
```
Comment (heitorlessa) [inline docs/core/event_handler/api_gateway.md:1037]: Shall we update to poetry pyproject.tomo since we're gonna update the sample cookiecutter already?

**Review by heitorlessa — CHANGES_REQUESTED** [review]
One minor change as we don't build w Lambda Layer for ARM yet, and one question

**Comment by heitorlessa** [comment]
Thanks a lot for helping revert to the previous sample

**Comment by michaelbrewer** [comment]
But it works. ;-)

> One minor change as we don't build w Lambda Layer for ARM yet, and one question

**Comment by heitorlessa** [comment]
This is what puzzles me hahaha

File: docs/core/event_handler/api_gateway.md (hunk 1 above)
Comment (michaelbrewer) [reply docs/core/event_handler/api_gateway.md:1076]: making it `x86_64` for now :)

**Comment by heitorlessa** [comment]
On a more serious note though, Tracer uses X-Ray SDK which depends on wrapt that uses C bindings.

Unless wrapt is giving a wheel for all platforms - like Pydantic - instead of the platform in question, then in theory Tracer shouldn't work.

All else should work as we don't depend on C bindings but pure Python.

File: docs/core/event_handler/api_gateway.md (hunk 2 above)
Comment (michaelbrewer) [reply docs/core/event_handler/api_gateway.md:1037]: @heitorlessa - currently I like `Pipefile` as this is the default for SAM cli and Amplify templates. However I am cool to switch to Poetry

**Comment by michaelbrewer** [comment]
@heitorlessa - updated.

File: docs/core/event_handler/api_gateway.md (hunk 2 above)
Comment (heitorlessa) [reply docs/core/event_handler/api_gateway.md:1037]: So let's keep as-is until we migrate others.

**Review by heitorlessa — APPROVED** [review]

**Pull request merged**